app/src/org/commcare/activities/QueryRequestUiController.kt

```diff
@@ -407,7 +411,43 @@ class QueryRequestUiController(
         return Localizer.processArguments(displayData.name, arrayOf("")).trim { it <= ' ' }
     }
 
+    private fun setHintPlaceholder(promptView: View, queryPrompt: QueryPrompt) {
+        val hintText = getHintText(queryPrompt)
+        if (hintText != "") {
+            val promptHelpButton = promptView.findViewById<View>(R.id.prompt_hint_button)
+            promptHelpButton.visibility = View.VISIBLE
+
+            promptHelpButton.setOnClickListener {
+                AlertDialogWrapper.alertDialog(queryRequestActivity, "Hint", hintText)
```

Comment:
Curios what you think about removing the wrapper and directly calling the `StandardAlertDialog.getBasicAlertDialog()` method from here.